backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserMutationResolver.java

```diff
@@ -57,10 +52,21 @@ public User updateUser(
     return new User(user);
   }
 
-  public OrganizationRole updateUserRole(UUID id, OrganizationRole role) {
+  public ApiOrganizationRole updateUserRole(UUID id, ApiOrganizationRole role) {
     return _us.updateUserRole(id, role);
   }
 
+  public User updateUserPrivileges(UUID id, 
+                                   boolean accessAllFacilities, 
+                                   List<String> facilities, 
```

Comment:
frankly, all the String facility id stuff is because when I was testing at one point using the graphql playground, the UUIDs that i was passing in in a list weren't getting auto-bound to UUIDs, so i gave up and made everything strings. but i can give it another try before writing it off.